kubernetes/client/rest.py

```diff
@@ -77,6 +77,9 @@ def __init__(self, configuration, pools_size=4, maxsize=None):
         if configuration.retries is not None:
             addition_pool_args['retries'] = configuration.retries
 
+        if configuration.tls_server_name:
```

Comment:
Unfortunately I think this file is generated, in order to make permanent changes here I believe the PR would need to go to the generator repo

Reply:
Can you provide some more information on what to do? Because the gen repo doesn't really say nor point to what needs to be updated. The contributing guide doesn't go into detail either.

This file seems to ref https://github.com/kubernetes-client/gen/blob/master/proto/dependencies.sh even more generated files from the apimachinery repo.

Reply:
I did a bit of digging, and the rest.py file appears to come from the openapi-generator upstream code:

https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/python/rest.handlebars

Would it be possible to implement a custom template per the upstream openapi-generator docs? https://openapi-generator.tech/docs/templating

Reply:
@fabianvf @yliaog 

The auto-generated code is available for `openapi-generator` version `>=6.6.0` ([PR](https://github.com/OpenAPITools/openapi-generator/pull/15283)).

Since this repo uses version `v4.3.0`, I decided to apply a patch - similar to the current one that exists - to apply the necessary changes to support SNI. Once we start using a version that includes the upstream fix, the patch can be safely removed.

Can you please check it again?

Reply:
thanks for the PR. it looks good to me, @fabianvf do you have any question or concern?